Tests/Rules/UseConsistentWhitespace.tests.ps1

```diff
@@ -388,4 +389,46 @@ if ($true) { Get-Item `
         }
     }
 
+
+    Context "CheckParameter" {
+        BeforeAll {
+            $ruleConfiguration.CheckInnerBrace = $true
+            $ruleConfiguration.CheckOpenBrace = $false
+            $ruleConfiguration.CheckOpenParen = $false
+            $ruleConfiguration.CheckOperator = $false
+            $ruleConfiguration.CheckPipe = $false
+            $ruleConfiguration.CheckSeparator = $false
+            $ruleConfiguration.CheckParameter = $true
+        }
+
+        It "Should not find no violation if there is always 1 space between parameters except when using colon syntax" {
```

Comment:
Some extra test scenarios it might be worth looking at, both in cases where we expect no diagnostic and ones where we do:

- Parameters mixed with splatting: `cmd -x a @var -y b`
- Parameters chopped down over lines with backticks:

```powershell
cmd -x a `
    -y b `
    -switch
```
- Parameters partially chopped down over lines

```powershell
cmd -switch `
    -x a -y b `
    -z c
```